{"name": "用户请求登陆 邮箱格式异常", "status": "passed", "steps": [{"name": "用户请求登陆", "status": "passed", "start": 1673432671538, "stop": 1673432671931}], "attachments": [{"name": "log", "source": "ff9f3551-c1f2-42fc-908a-2a6094c9563f-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "test_title", "value": "'用户请求登陆 邮箱格式异常'"}, {"name": "email", "value": "''"}, {"name": "status_code", "value": "400"}, {"name": "code", "value": "2100000"}], "start": 1673432671538, "stop": 1673432671931, "uuid": "861f3513-9416-4024-b90d-4bf987a40753", "historyId": "a7a3e4d5abeb9090236bd6a900b72cc7", "testCaseId": "90ed5c9f8363914f77eeda20f66704fa", "fullName": "Testcase.Test_User.test_user_fail.Test_create_custodial_account#test_create_account_custodial", "labels": [{"name": "feature", "value": "Users Fail!"}, {"name": "story", "value": "User Connect Fail!"}, {"name": "parentSuite", "value": "Testcase.Test_User"}, {"name": "suite", "value": "test_user_fail"}, {"name": "subSuite", "value": "Test_create_custodial_account"}, {"name": "host", "value": "lilongdeMacBook-Pro.local"}, {"name": "thread", "value": "2505-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Testcase.Test_User.test_user_fail"}]}